lib/util/cmcd_manager.js

```diff
@@ -388,6 +390,29 @@ shaka.util.CmcdManager = class {
     }
   }
 
+  /**
+   * Filter the CMCD data object to include only the keys specified in the
+   * configuration.
+   *
+   * @param {CmcdData} data
+   * @return {CmcdData}
+   * @private
+   */
+  filterKeys_(data) {
+    const includeKeys = this.config_.includeKeys;
+
+    if (!includeKeys || !includeKeys.length) {
```

Comment:
This will never be null.  The default is an empty array, and configure() will type-check inputs and reject null.

Reply:
Done.